src/file_name_plugin/__tests__/plugin.test.js

```diff
@@ -110,3 +139,32 @@ it('file matching is case insensitive', async () => {
   type(KEYS.ENTER);
   await runPromise;
 });
+
+it("selected file doesn't include trimming dots", async () => {
+  const { hookEmitter, updateConfigAndRun, plugin, type } = pluginTester(
+    FileNamePlugin,
+    {
+      stdout: { columns: 40 },
+    },
+  );
+
+  hookEmitter.onFileChange({
+    projects: [
+      {
+        config: {
+          rootDir: '/project',
+        },
+        testPaths: ['/project/src/long_name_gonna_need_trimming.js'],
+      },
+    ],
+  });
+  const runPromise = plugin.run({}, updateConfigAndRun);
+
+  type('t', 'r', 'i', 'm', 'm', KEYS.ARROW_DOWN, KEYS.ENTER);
+  await runPromise;
+
+  expect(updateConfigAndRun).toHaveBeenCalledWith({
+    mode: 'watch',
+    testPathPattern: 'ing\\.js',
```

Comment:
I believe this should have included more - width is set to 40 after all. But this is a separate issue that could be investigated.